boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -145,3 +154,17 @@
 &gpio4{
 	status = "okay";
 };
+
+&flexcan2 {
+	pinctrl-0 = <&flexcan2_default>;
+	pinctrl-names = "default";
+	sample-point = <875>;
+	sample-point-data = <875>;
+	bus-speed = <125000>;
+	bus-speed-data = <1000000>;
+	phys = <&flexcan2_phy>;
+	status = "disabled";
+	can-transceiver {
+		max-bitrate = <8000000>;
+	};
```

Comment:
You cannot have both the `phys` property and a `can-transceiver` child node.

Reply:
removed.